src/merge.py

```
@@ -83,68 +83,68 @@ def validate_2fa(api: PyiCloudService) -> bool:
 
     return status
 
 def get_datetime(dt:datetime) -> datetime:
     return datetime(dt.year, dt.month, dt.day, dt.hour, dt.minute, tzinfo=dt.tzinfo)
 
-def get_from_list(items, value:str):
+def get_from_list(items:list, value:str):
     try:
         return_value = items.get(value)
-    except (AttributeError, TypeError):
+    except:
```

Ruff (0.15.9)

[error] Do not use bare `except`

(E722)